generators/newapp/new.go

```diff
@@ -98,12 +99,61 @@ func (a Generator) Run(root string, data makr.Data) error {
 		},
 	})
 
+	gn, err := a.genny()
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	g.Add(gn)
+
 	a.setupVCS(g)
 
 	data["opts"] = a
 	return g.Run(root, data)
 }
 
+func (a Generator) genny() (makr.Runnable, error) {
+	app := a.App
+	run := genny.WetRunner(context.Background())
```

Comment:
@markbates I noticed the current development branch generates Dockerfile and .dockerignore outside the project directory. 

For example, this generates the Dockerfiles in `../petstore`
```
buffalo new petstore --api -f
```

I tracked this down to simply adding a line after this `run.Root = app.Root`, but I'm not entirely sure this is idiomatic. I'm thinking WetRunner should take in a Path as a variable. 

What do you think?